flask_of_oil/jwt_validator.py

```diff
@@ -33,13 +33,15 @@ class JwtValidatorException(Exception):
 
 class JwtValidator:
     def __init__(self, jwks_url, issuer, audience, verify_ssl_server=True):
+        self.logger = logging.getLogger(__name__)
         self.supported_algorithms = ['RS256', "RS512"]
         self.jwks_url = jwks_url
         self.aud = audience
         self.iss = issuer
         self.verify_ssl_server = verify_ssl_server
+        self.lastSuccessfulJWKSFetch = None
```

Comment:
should prob initialize to `0` to not have the possibility for the exception mentioned below